docs/run-basic.md

```diff
@@ -0,0 +1,93 @@
+# Running the OMS Locally
+
+Follow these instructions to run the OMS locally, using the 
+Temporal Service provided that the `temporal` command. This 
+is an expanded version of the instructions found in the 
+_Quickstart_ section of the top-level README file.
+
+
+### Start the Temporal Service
+
+Run the following command in your terminal to start the Temporal 
+Service:
+
+```command
+temporal server start-dev \
+    --ui-port 8080 \
+    --db-filename temporal-persistence.db
+```
+
+The `temporal` CLI provides a convenient way of running a 
+Temporal Service locally for development purposes. By default,
+it provides a Web UI on port 8233 and persists data to an 
+ephemeral in-memory database. The options in this command 
+change the Web UI port to 8080 and instructs the Temporal 
+Service to persist its data to a file so that it will be 
+available in subsequent sessions. This file will be created 
+if it does not exist.
+
+
+### Start the Worker
+
+Run the following command in another terminal to start the Workers:
+
+```command
+go run ./cmd/oms worker
+```
+
+The Workers run the Workflow and Activity code used to implement the
+OMS. Although one Worker is sufficient for local development, Temporal
+recommends running multiple Workers in production since this can improve
+both the scalability and availability of an application. You can repeat
+this step to launch as many additional Workers as you like.
```

Comment:
```suggestion
this step to launch as many additional Workers as you like.  Temporal's SDK will automatically distribute processing load between all the running workers.
```

Reply:
Addressed via [1489713](https://github.com/temporalio/reference-app-orders-go/commit/14897138001add577ea4d12f38a801cc2c5e4f30), but changed "between" to "among" since there would likely be more than two Workers.